server/src/entities/user.entity.ts

```diff
@@ -53,11 +53,13 @@ export class User extends BaseEntity {
 
   public isAdmin;
   public isDeveloper;
+  public role;
 
   @AfterLoad()
   computeUserRole(): void {
     this.isAdmin = this.organizationUsers[0].role === 'admin';
     this.isDeveloper = this.organizationUsers[0].role === 'developer';
+    this.role = this.organizationUsers[0].role;
```

Comment:
Does it make sense to make do with only with `role` and rid of `isAdmin` and `isDeveloper` as it would be redundant here?